providers/implementations/keymgmt/ml_kem_kmgmt.c

```diff
@@ -334,16 +377,32 @@ static const OSSL_PARAM *ml_kem_settable_params(void *provctx)
 static int ml_kem_set_params(void *vkey, const OSSL_PARAM params[])
 {
     ML_KEM_KEY *key = vkey;
-    const ML_KEM_VINFO *v = ossl_ml_kem_key_vinfo(key);
     const OSSL_PARAM *p;
-    const void *pubenc = NULL;
-    size_t publen = 0;
+    const void *pubenc = NULL, *prvenc = NULL;
+    size_t publen = 0, prvlen = 0;
 
     if (ossl_param_is_empty(params))
         return 1;
 
-    p = OSSL_PARAM_locate_const(params, OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY);
-    if (p == NULL)
+    p = OSSL_PARAM_locate_const(params, OSSL_PKEY_PARAM_ENCODED_PRIVATE_KEY);
```

Comment:
I did consider using this name when I was doing other algorithms, Not sure if we should just be using PRIV_KEY here instead like we do in other places.

Reply:
The `get` and `set` interfaces work with *encoded* keys, while the `export` and `import` interfaces work with raw keys.